lib/navigation/view/scaffold.dart

```diff
@@ -139,11 +141,16 @@ class AppScaffoldAction {
                   .toList();
             },
           )
-        : TextButton.icon(
-            label: Text(text ?? tooltip ?? ''),
-            icon: icon,
-            onPressed: onPressed,
-          );
+        : (tooltip == null)
+            ? TextButton(
```

Comment:
This is a bit weird. If there's no tooltip, we don't show an icon either?

Reply:
I didn't check all the usages, the issue was on the `Save` buttons which had none. I figured ActionButtons only have tooltips when they also have an icon

Reply:
I updated it and checked only if the icon exists